src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/Value.tsx

```diff
@@ -33,6 +34,8 @@ const EtherscanLink = ({ method, type, value }: RenderValueProps): React.ReactEl
   const [cut, setCut] = React.useState(undefined)
   const { width } = useWindowDimensions()
 
+  const etherscanType: paramType = ['tx', 'address'].includes(type) ? (type as paramType) : 'address'
```

Comment:
Hey, just some semantic changes.

I was having trouble to read this line at first sight. What do you think about calling `etherscanType` to the generic one instead of `paramType` and `linkType` to the inner one instead of `etherscanType`?

Reply:
Sounds good. Done!

Reply:
I don't get this check. The type says it allows string | 'address' | 'tx'. Then we check if passed type is 'address' or 'tx' and if not, we set it to 'address'
1. Why do we allow string then?
2. Why do we need this check? I would change the type to 'address' | 'tx' and let the typescript catch any errors.

Reply:
Because it comes from a parent component and that one handle more than 2 types but for the etherscan link need to be either `address` or `tx`. If not, with a `bool` type, we wouldn't have an etherscan link.

Reply:
I took a brief look and the type prop doesn't refer to etherscan type parameter, this is type of a function argument. `EtherscanLink` component is rendered only if the parameter is of address type:
```
  if (isAddress(type)) {
    return <EtherscanLink type={type} {...props} />
  }
```

So in this case the type of a link should be always `address`, I fixed it in my commit